src/Mod/Assembly/JointObject.py

```diff
@@ -565,6 +583,24 @@ def onChanged(self, joint, prop):
         """Do something when a property has changed"""
         # App.Console.PrintMessage("Change property: " + str(prop) + "\n")
 
+        # if not hasattr(joint, "MissingReference"):
+            # self.addMissingRefProperty(joint)
+
```

Comment:
MissingRef should not be a property. You do not need to store this. What is the point of storing it anyway?

You should check in AssemblyObject.cpp `if (joint.Reference1[1][0].find("?") != -1) or (joint.Reference2[1][0].find("?") != -1):` or something like this.